packages/driver/test/cypress/integration/commands/actions/type_spec.js

```diff
@@ -2822,6 +3018,10 @@ describe('src/cy/commands/actions/type', () => {
           expect($input).to.have.value('FoO')
         })
       })
+
+      it('{shift} does not capitalize characters', () => {
+        cy.get('input:first').type('{shift}foo').should('have.value', 'foo')
```

Comment:
likely need a note here to describe our implementation logic of why we consider this to be correct